From 59901632d6c557187d4c375890b92a8ef1aa0c82 Mon Sep 17 00:00:00 2001
From: Sunil Goutham <sgoutham@marvell.com>
Date: Tue, 6 Nov 2018 10:40:50 +0530
Subject: [PATCH 0748/1051] soc: octeontx2: Fix npa_pf_func size in
 sso_hw_setconfig mbox msg

PF_FUNC should be 16bit and not 32bit.

Signed-off-by: Sunil Goutham <sgoutham@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/soc/marvell/octeontx2/mbox.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/soc/marvell/octeontx2/mbox.h b/drivers/soc/marvell/octeontx2/mbox.h
index e5a62119b7f2..793a8f03aa7d 100644
--- a/drivers/soc/marvell/octeontx2/mbox.h
+++ b/drivers/soc/marvell/octeontx2/mbox.h
@@ -707,7 +707,7 @@ struct ssow_lf_free_req {
 struct sso_hw_setconfig {
 	struct mbox_msghdr hdr;
 	u32	npa_aura_id;
-	u32	npa_pf_func;
+	u16	npa_pf_func;
 	u16	hwgrps;
 };
 
-- 
2.17.1

